transformations/concept2sentence/transformation.py

```diff
@@ -0,0 +1,70 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+from sibyl import Concept2Sentence
+
+
+class C2S(SentenceAndTargetOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION
+    ]
+    languages = ["en"]
+
+    def __init__(self, 
+                 seed=0, 
+                 max_outputs=1, 
+                 dataset=None, 
+                 extract="token",
+                 gen_beam_size=10,
+                 text_min_length=10,
+                 text_max_length=32,
+                 device='cpu',
+                 task_config=None):
+
+        super().__init__(seed, max_outputs=max_outputs)
+
+        self.c2s = Concept2Sentence(
+                        dataset = dataset,
+                        extract = extract,
+                        gen_beam_size = gen_beam_size,
+                        text_min_length = text_min_length,
+                        text_max_length = text_max_length,
+                        device = device)
+        self.task_config = task_config
+        if self.task_config is None:
+            self.task_config = {
+                'input_idx': [1],
+                'tran_type': 'INV',
+                'label_type': 'hard',
+                'task_name': 'topic'
+            }
+
+    def generate(self, sentence: str, target: int):
+        new_sentence, new_target = self.c2s.transform_Xy(sentence, target, self.task_config)
+        return new_sentence[0], new_target
```

Comment:
This should generate a list of pairs or a list of tuples if I'm not wrong.

Reply:
Corrected